src/test/java/com/wikia/webdriver/common/core/SEOUtils.java

```diff
@@ -106,4 +122,17 @@ public boolean isLinkRelCanonical() throws WebDriverException {
     }
     return driver.getCurrentUrl().equals(canonicalUrl.getAttribute("href"));
   }
+
+  public boolean isAttributesListPresentInRobotsMetaTag(List<String> expectedAttributes) {
+    String[] currentAttributes = robots.getAttribute("content").split("[, ]+");
+
+    if (currentAttributes.length != expectedAttributes.size()) {
+      PageObjectLogging.log(
+          "isAttributesListPresentInRobotsMetaTag",
+          "Number of current attributes is different than expected",
+          false);
```

Comment:
well because false in PageObjectLogging.log is really causing test to fail I would consider either renaming method to assert or returning true or false, logging here what content was passed to method and what content was found on page